src/bindings/rust/README.md

```diff
@@ -8,30 +8,102 @@
 
 Rust bindings for libelektra.
 
+## Build
+
+To build the bindings explicitly as part of the elektra build process, we add the option `rust` to `-DBINDINGS`. Now [build libelektra](../../../doc/COMPILE.md) and the bindings will be built as part of this process.
+
+## Example
+
+Note that your dynamic linker must be able to find `libelektra`. If you just compiled it, you can run `source ../scripts/run_dev_env` from the `build` directory to modify your `PATH` appropriately.
+
+See the `example` directory for a fully setup project. To run it, change directories into `build/src/bindings/rust/example/` and run `cargo run`.
+
+To start with a new project, use `cargo new elektra_rust`. Now add the `elektra` crate to the dependencies. The crate is in the `src/bindings/rust` subdirectory of your `build` directory, so the exact paths depends on your system. Change the paths (and possibly version) appropriately and add the following dependencies to your `Cargo.toml`.
+
+```toml
+[dependencies]
+elektra = { version = "0.9.0", path = "~/git/libelektra/build/src/bindings/rust/elektra" }
+# Directly depending on elektra-sys is only needed if you need to use the raw bindings
+elektra-sys = { version = "0.9.0", path = "~/git/libelektra/build/src/bindings/rust/elektra-sys" }
+```
+
+If you run `cargo run` and everything builds correctly and prints `Hello, world!`, you can replace the contents of `main.rs` with the examples shown in the next section.
+
 ## Usage
 
 ### Raw Bindings
```

Comment:
I would move this to the end. So first key, keyset, kdb and then about raw bindings.